torch/csrc/jit/script/compiler.cpp

```diff
@@ -1130,8 +1130,12 @@ struct to_ir {
             }
           }
         }
-        return CondValue(
-            emitToBool(emitExpr(expr)), RefinementSet({}), c10::nullopt);
+        auto expr_out = emitToBool(emitExpr(expr));
```

Comment:
For robustness, I'd prefer this looked at the node before emitExpr to determine whether it is an is_scripting call. Doing it afterward is subject to arbitrary peephole optimizations that code emission might do and doesn't match the pattern in the rest of this function.

Reply:
The AST looks like:
```
    (apply
      (.
        (.
          (variable (ident torch))
          (ident jit))
        (ident is_scripting))
      (list)
      (list)))
```
How should I pattern match then? Are we taking the stance that only `torch.jit.is_scripting` works, and not `jit.is_scripting`, or `is_scripting`, or 
```
import torch.jit.is_scripting as COMPILE_BLOCK
if not COMPILE_BLOCK()
...
```

i'm fine with whatever. up to this point we have only used condvalues with builtin reserved keywords, so this is a different case.